### ISSUE TITLE
Not updating the Meetup Talk Descriptions

### ISSUE THREAD

**Issue by robinhilliard** [elixirsydney/elixirsydney#32]
Hi, can we update the talk descriptions on Meetup when we receive them? The last few talks I've done the invite that goes out from Meetup still says looking for talks, even on the day of the event. I can't help but think that it impacts people's interest in attending. Do you need to delegate to someone? They will need Meetup authentication.

**Comment by paulfioravanti** [comment]
I think I dropped the ball on that this month, so apologies for that.

I try and update the meetup descriptions as talks come in, but rounding up talks is easily the most difficult part of managing a meetup, and there are many instances where talks proposals come through, or get cancelled, close to the meetup date. Unfortunately, Meetup blasts out emails at a set number of days before the scheduled date; this setting can't be changed, and in some cases we will still be trying to get speakers, so you'll see the message of looking for talks.

I've tried to mitigate this somewhat by putting a link to the issues page so if the meetup page hasn't been updated, then the advice is to go check there, but if this is still a source of confusion, then we'll have to re-visit it.
